src/rust/src/rdatatype.rs

```diff
@@ -361,6 +361,24 @@ pub fn robj_to_interpolation_method(robj: Robj) -> RResult<pl::InterpolationMeth
     }
 }
 
+pub fn robj_to_cloud_options<'a>(
+    url: &'a str,
+    robj: &'a Robj,
+) -> RResult<Option<pl::cloud::CloudOptions>> {
```

Comment:
Not expert on Rust but are those lifetimes necessary here? Maybe it's better practice, but for consistency it would be nice to remove them to match the other `robj_to_*` functions so that later we don't wonder why there are lifetimes only for this function.